.eslintignore

```diff
@@ -0,0 +1,5 @@
+**/node_modules/*
```

Comment:
Include all your config requirements (i.e. .eslintignore, .eslintrc, .gitignore, etc.) in the root of your same file you're running your lab. Right now, the placement of these docs in the root of the folder, ``lab-06-wack-chat`` when they're better placed inside ``lab-devon.``